and-bible/app/src/main/AndroidManifest.xml

```diff
@@ -4,11 +4,12 @@
       android:installLocation="auto"
       android:versionName="2.11.0.245" android:versionCode="245">
 
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
```

Comment:
I think there is a way of declaring that another app can freely access your apps storage, which would be nice to give to other Sword apps like And Bible Addons.  However, I can't find the technique - I will keep looking - unless you find it first.

Reply:
I did not find anything like that. But if And Bible Addons requests READ_EXTERNAL_STORAGE, then it can save modules into our folder.